src/app/components/authorized/add-new-post/add-new-post.component.scss

```diff
@@ -5,9 +5,11 @@ form {
     @include form;
 }
 
-.typeButton:focus{
-    color: $orange;
-}
-.typeButton{
-    background-color:lightblue;
+.gridWrapper {
```

Comment:
This feels weird to me as you have `gridWrapper` and `wrapper` which both have `grid` display. I'm not a "CSS master" though, so this may be totally fine. The naming feels odd though.